device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_config.json

```diff
@@ -0,0 +1,326 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "phy1",
+      "address": "1",
+      "lib_name": "/usr/lib/libsai_credo.so",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy1_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "hwinfo": "mdio0_0_0/0"
+    },
+    {
+      "phy_id": 2,
+      "name": "phy2",
+      "address": "2",
+      "lib_name": "/usr/lib/libsai_credo.so",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy2_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "hwinfo": "mdio1_0_0/0"
+    },
+    {
+      "phy_id": 3,
+      "name": "phy3",
+      "address": "3",
+      "lib_name": "/usr/lib/libsai_credo.so",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy3_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "hwinfo": "mdio2_0_0/0"
+    },
+    {
+      "phy_id": 4,
+      "name": "phy4",
+      "address": "4",
+      "lib_name": "/usr/lib/libsai_credo.so",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy4_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "hwinfo": "mdio3_0_0/0"
+    },
+    {
+      "phy_id": 5,
+      "name": "phy5",
+      "address": "5",
+      "lib_name": "/usr/lib/libsai_credo.so",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy5_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "hwinfo": "mdio4_0_0/0"
+    },
+    {
+      "phy_id": 6,
+      "name": "phy6",
+      "address": "6",
+      "lib_name": "/usr/lib/libsai_credo.so",
```

Comment:
is this really used? i thought the name is libsai.so.1?

Reply:
No, not really used

Reply:
if not used, can we remove it?

Reply:
It has been removed.